### PR TITLE
Added clearData to the reporting.py to clear report files from DB after processing them

### PR DESCRIPTION
**Pull request by aydarng**
**Verifier Service Release Notes**

**Running the Verifier Service**

**Configuration:**  
To run the Verifier service, you need to specify a configuration file by providing the following arguments:  
`--config-dir=scripts --config-file='your_config_file'`

**Configuration Files Location:**  
Configuration files are located in the following directory:  
`scripts/keri/cf`

**Configurable Parameters:**  
You can customize the service behavior using these options:

-   **iurls**: OOBI URLs
-   **durls**: Schema OOBI URLs
-   **trustedLeis**: A list of trusted LE identifiers.
-   **allowedEcrRoles**: Roles permitted for ECR credential authorization.
-   **allowedOorRoles**: Roles permitted for OOR credential authorization.
-   **allowedSchemas**: A list of schemas allowed for authorization.

**Environment Variables:**  
You can export the following environment variables to modify runtime behavior:

-   **VERIFIER_ENV**: Set to `dev` or `production`. When set to `production`, the `/root_of_trust` endpoint is disabled. Default: `production`.
-   **VERIFY_ROOT_OF_TRUST**: Set to `True` or `False`. Enables or disables root of trust validation logic. Default: `True`.
-   **KERI_BASER_MAP_SIZE**: Defines the maximum size for the LMDB database. Default: `104857600` (100 MB).

**Example:**  
Run the Verifier service with the following command:  
`verifier server start--config-dir=scripts --config-file='verifier-config-public.json'`

----------

**LMDB Reports Data Cleanup**

The Verifier now includes automatic cleanup of reports after processing:

-   Processed reports are removed from the LMDB database to prevent size issues.
-   This update addresses crashes caused by the LMDB table exceeding the size defined by `KERI_BASER_MAP_SIZE`.

----------

**Bug Fixes**

-   Resolved an issue where uploading multiple reports caused the Verifier to crash with the error:  
    `lmdb.MapFullError: mdb_put: MDB_MAP_FULL: Environment mapsize limit reached`

----------

**Summary**  
This release introduces:

-   Configurable parameters for better customization.
-   Support for environment variables to adapt to different environments.
-   Automatic LMDB cleanup to enhance stability and prevent size-related crashes.

### CODE REVIEW

**Review by 2byrds — APPROVED** [review]
LGTM

**Review by 2byrds — APPROVED** [review]
lgtm

**Review by 2byrds — APPROVED** [review]
lgtm

**Comment by aydarng** [comment]
**Verifier Release Notes**

**Running the Verifier Service**

**Configuration:**  
To run the Verifier service, you need to specify a configuration file by providing the following arguments:  
`--config-dir=scripts --config-file='your_config_file'`

**Configuration Files Location:**  
Configuration files are located in the following directory:  
`scripts/keri/cf`

**Configurable Parameters:**  
You can customize the service behavior using these options:

-   **iurls**: OOBI URLs
-   **durls**: Schema OOBI URLs
-   **trustedLeis**: A list of trusted LE identifiers.
-   **allowedEcrRoles**: Roles permitted for ECR credential authorization.
-   **allowedOorRoles**: Roles permitted for OOR credential authorization.
-   **allowedSchemas**: A list of schemas allowed for authorization.

**Environment Variables:**  
You can export the following environment variables to modify runtime behavior:

-   **VERIFIER_ENV**: Set to `dev` or `production`. When set to `production`, the `/root_of_trust` endpoint is disabled. Default: `production`.
-   **VERIFY_ROOT_OF_TRUST**: Set to `True` or `False`. Enables or disables root of trust validation logic. Default: `True`.
-   **KERI_BASER_MAP_SIZE**: Defines the maximum size for the LMDB database. Default: `104857600` (100 MB).

**Example:**  
Run the Verifier service with the following command:  
`python verifier.py --config-dir=scripts --config-file='verifier-config-public.json'`

----------

**LMDB Reports Data Cleanup**

The Verifier now includes automatic cleanup of reports after processing:

-   Processed reports are removed from the LMDB database to prevent size issues.
-   This update addresses crashes caused by the LMDB table exceeding the size defined by `KERI_BASER_MAP_SIZE`.

----------

**Bug Fixes**

-   Resolved an issue where uploading multiple reports caused the Verifier to crash with the error:  
    `lmdb.MapFullError: mdb_put: MDB_MAP_FULL: Environment mapsize limit reached`

----------

**Summary**  
This release introduces:

-   Configurable parameters for better customization.
-   Support for environment variables to adapt to different environments.
-   Automatic LMDB cleanup to enhance stability and prevent size-related crashes.

File: src/verifier/core/reporting.py [hunk 1]
```diff
@@ -107,7 +106,7 @@ def create(self, aid: str, dig: str, lei: str, filename: str, typ: str, stream):
         diger = DigerBuilder.sha256(dig)
         report = b''
         while True:
-            chunk = stream.read(4096)
+            chunk = stream.read(4194304)
```
Comment (2byrds) [inline src/verifier/core/reporting.py:109]: Do you want to do os.getenv("FILER_CHUNK_SIZE") ?

**Review by 2byrds — APPROVED** [review]
Do you want to keep the release notes in a file in the repo and reference it from the README?

File: src/verifier/core/reporting.py (hunk 1 above)
Comment (aydarng) [reply src/verifier/core/reporting.py:109]: Sure, I'll do this way. And I'll make default value to be 4194304(4MB)

**Comment by aydarng** [comment]
> Do you want to keep the release notes in a file in the repo and reference it from the README?

Added CHANGELOG.md

**Pull request merged**